SConstruct

```diff
@@ -1412,8 +1412,13 @@ else :
 	pythonEnv["PYTHON_INCLUDE_FLAGS"] = ""
 
 pythonEnv.Append( CPPFLAGS = "-DBOOST_PYTHON_MAX_ARITY=20" )
+
+boostPythonLibSuffix = ""
+if int( env["BOOST_MAJOR_VERSION"] ) >= 1 and int( env["BOOST_MINOR_VERSION"] ) >= 67 :
```

Comment:
This would return `False` if we had a version 2.0 of Boost.
Maybe you expect that by then you'd get rid of this line completely?
Otherwise something like the following example would be safer:
```
if ( int( env["BOOST_MAJOR_VERSION"] ) == 1 and int( env["BOOST_MINOR_VERSION"] ) >= 67 ) or int( env["BOOST_MAJOR_VERSION"] ) > 1 :
```

Reply:
Good catch - thanks. Fixed in 866e550.